grpc/src/main/java/com/linecorp/armeria/internal/client/grpc/ArmeriaClientCall.java

```diff
@@ -252,7 +258,13 @@ public void start(Listener<O> responseListener, Metadata metadata) {
                 deframed.subscribe(this, ctx.eventLoop(), SubscriptionOption.WITH_POOLED_OBJECTS);
             });
         }
-        responseListener.onReady();
+        contextAwareExecutor.execute(() -> {
+            try {
+                responseListener.onReady();
```

Comment:
This is a new behavior: we now wrap any user-defined callback with try-catch and close if anything throws. Previously, we've only done it with onMessage, but user code can be injected as interceptor so any callback on responseListener should be treated as "untrusted".